Makefile

```diff
@@ -5,6 +5,9 @@ DOCKERBUILD := docker build --build-arg http_proxy=$$http_proxy --build-arg http
 
 PIP?=pip
 
+VENV_NAME:=.venv$(VENV_PY_VERSION)
```

Comment:
problem with this is that the de-facto standard for venv is .venv, which you can configure your editor to automatically find.
So I would prefer to have default python venv use ".venv"